packages/rizin/build.sh

```diff
@@ -0,0 +1,46 @@
+TERMUX_PKG_HOMEPAGE=https://rizin.re
+TERMUX_PKG_DESCRIPTION="UNIX-like reverse engineering framework and command-line toolset"
+TERMUX_PKG_LICENSE="LGPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=v0.6.2
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	# Certain packages are not safe to build on device because their
+	# build.sh script deletes specific files in $TERMUX_PREFIX.
+	if $TERMUX_ON_DEVICE_BUILD; then
+		termux_error_exit "Package '$TERMUX_PKG_NAME' is not safe for on-device builds."
+	fi
+}
+
+# Get the Android architecture
+ARCH=$(getprop ro.product.cpu.abi)
+
+# Set the appropriate source URL based on the architecture
+if [ "$ARCH" == "arm64-v8a" ]; then
+    TERMUX_PKG_SRCURL=https://github.com/rizinorg/rizin/releases/download/$TERMUX_PKG_VERSION/rizin-$TERMUX_PKG_VERSION-android-aarch64.tar.gz
```

Comment:
This appears to be a precompiled binary.
Please the source code (`rizin-src-${TERMUX_PKG_VERSION}.tar.xz`) archive instead as packages need to be compiled for the `$TERMUX_PREFIX`, and as such have to be built from source.

---

Alternatively, since the linked archive already appears to be the Android version of Rizin, use the standalone app.